local-interchain/interchain/router/router.go

```diff
@@ -36,6 +39,27 @@ func NewRouter(
 	infoH := handlers.NewInfo(config, installDir, ctx, ic, cosmosChains, vals, relayer, eRep)
 	r.HandleFunc("/info", infoH.GetInfo).Methods(http.MethodGet)
 
+	wd, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+
+	chainRegistryFile := filepath.Join(wd, "chain_registry.json")
+	if _, err := os.Stat(chainRegistryFile); err == nil {
+		crH := handlers.NewChainRegistry(chainRegistryFile)
+		r.HandleFunc("/chain_registry", crH.GetChainRegistry).Methods(http.MethodGet)
+	} else {
+		log.Printf("chain_registry.json not found in %s, not exposing endpoint.", wd)
+	}
+
+	chainRegistryAssetsFile := filepath.Join(wd, "chain_registry_assets.json")
+	if _, err := os.Stat(chainRegistryAssetsFile); err == nil {
+		crH := handlers.NewChainRegistry(chainRegistryAssetsFile)
+		r.HandleFunc("/chain_registry_assets", crH.GetChainRegistry).Methods(http.MethodGet)
+	} else {
+		log.Printf("chain_registry_assets.json not found in %s, not exposing endpoint.", wd)
+	}
+
```

Comment:
we call the same handler here so is it just handling the case where the file could be named either `chain_registry.json` or `chain_registry_assets`? is there any reason for this explicit handling of both?

Reply:
it uses the same as it just returns the JSON object. Since its specific to chain registry and only do that 1 function, it makes sense to just re-use it vs duplicate code